test_idref.py

```diff
@@ -0,0 +1,86 @@
+from invenio_db import db
```

Comment:
licence is missing, can you add a script to check?

Reply:
This file looks like a debug file is it really a part of this commit?

Reply:
deleted